objmodel/chapter.md

```diff
@@ -885,44 +877,44 @@ class Instance(Base):
             self.map = new_map
 ````
 
-The class now passes ``None`` as the fields dict to ``Base``, as ``Instance``
-will now store the content of the dict in another way. Therefore it now needs
-to override the the ``_read_dict`` and ``_write_dict`` methods. In a real
-implementation, we would refactor the ``Base`` class to not be responsible for
-storing the field dictionary any more, but for this section having instances
+The class now passes ``None`` as the fields dictionary to ``Base``, as ``Instance``
+will store the content of the dictionary in another way. Therefore it needs
+to override the ``_read_dict`` and ``_write_dict`` methods. In a real
+implementation, we would refactor the ``Base`` class so that it is no longer responsible for
+storing the fields dictionary, but for now having instances
 store ``None`` there is good enough.
 
-When creating a new instance it starts out with using the ``EMPTY_MAP``, which
-has no attributes, and an empty storage. To implement ``_read_dict``, the
+When creating a new instance, the class starts out using the ``EMPTY_MAP``, which
```

Comment:
This is wrong. It's really the *instance* that starts out using the ``EMPTY_MAP``.